news.d/feature/1514.ui.md

```diff
@@ -0,0 +1 @@
+Read and apply Qt style sheet from file in config directory.
```

Comment:
This can probably be reworded to be clearer, and specify the name of the name of the file Plover will look for.